internal/ad/adsys-gpolist_test.go

```diff
@@ -227,7 +228,7 @@ func TestAdsysGPOList(t *testing.T) {
 			}
 
 			// #nosec G204: we control the command line name and only change it for tests
-			cmd := exec.Command(adsysGPOListcmd, "--objectclass", tc.objectClass, tc.url, tc.accountName)
+			cmd := exec.Command(adsysGPOListcmd, "--objectclass", tc.objectClass, tc.url, strings.ToLower(tc.accountName))
```

Comment:
why lower casing here and not the test input? That may make people thinking that this command will take non lowercase content, where we will only pass lowercase ones.